widgets/hits.js

```diff
@@ -16,6 +21,9 @@ function hits({container = null, templates = {}, hitsPerPage = 20}) {
           results={results}
           helper={helper}
           noResultsTemplate={templates.empty}
+          hideWhenNoResults={hideWhenNoResults}
+          emptyWhenNoResults={hideWhenNoResults}
```

Comment:
Because the behavior of autoHide is to empty by default when no results, while for the hits by default we do not want it to be empty when no results